reana-ui/src/components/WorkflowActionsPopup.js

```diff
@@ -92,7 +92,7 @@ export default function WorkflowActionsPopup({ workflow, className }) {
       content: "Free up disk",
```

Comment:
Technically we shouldn't need this option any more, but it might be useful to keep it for a while for users who have previously deleted the workflow without the workspace